api/institutions/views.py

```diff
@@ -1,11 +1,14 @@
 from rest_framework import generics
 from rest_framework import permissions as drf_permissions
+from rest_framework import status
+from rest_framework.response import Response
 
 from modularodm import Q
 
 from framework.auth.oauth_scopes import CoreScopes
 
-from website.models import Institution, Node, User
+from website.models import Node, User
+from website.project.model import Institution
```

Comment:
For consistency, I think it still makes sense to have `Institution` imported in `website.models`. You don't need to add it to `website.models.MODELS` because that includes all the StoredObjects that get attached to Mongo when we initialize the app.